### DISCUSSION TITLE
Can't leech/mirror torrent files

### DISCUSSION THREAD

**Discussion by paulper2** [anasty17/mirror-leech-telegram-bot#1321] [General]
I am using /ql command to leech torrent file its showing me NOTES only magnet links are working , I cant use /ql 10 cmd too if I try it on magnet links it again show me note I have attached the image below please check 
![t](https://user-images.githubusercontent.com/93882660/210272165-43f145d4-0a3a-4f34-b7de-1c35acb7e752.png)

**Comment by anasty17** [comment]
Promote ur bot in group to admin

**Reply by paulper2** [reply]
Did promoted bot to admin still same issue

**Reply by anasty17** [reply]
Resend the links and try

**Reply by paulper2** [reply]
Tysm its working...😁
